rawdb/rawdb.go

```diff
@@ -31,7 +31,10 @@ var (
 )
 
 func init() {
-	loggo.ConfigureLoggers(logLevel)
+	err := loggo.ConfigureLoggers(logLevel)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Should this be a panic to match the other init functions?

Reply:
yeah